components/sidebar/channel_navigator/channel_navigator.tsx

```diff
@@ -165,7 +164,7 @@ export default class ChannelNavigator extends React.PureComponent<Props> {
         );
 
         let layout;
-        if (isDesktopApp() && !this.props.globalHeaderEnabled) {
+        if (isDesktopApp()) {
```

Comment:
Shouldn't this whole block be removed if we're always turning on the global header now?

Edit: Yeah, it should be removed since we've got duplicated arrows now in the LHS.